minutes/2025-07-01.md

```diff
@@ -1,20 +1,47 @@
-## TSC Attendees
+# Hiero Technical Steering Committee Meeting
 
-## Guests
+**TL;DR:** Hendrik confirmed two valid nominations for the open TSC seat (May and Brandon), with voting to commence shortly. Members discussed election logistics, including tie scenarios and tooling. A proposal to create a dedicated Hiero Marketing Committee gained strong support. Final hurdles to project graduation (notably OpenSSF scorecard inclusion) were also reviewed.
 
-As every Hiero meeting the TSC meeting is a public meeting and everybody can attend it.
-You can find all the information about the TSC meetings and all other Hiero meetings in our [public calender](https://zoom-lfx.platform.linuxfoundation.org/meetings/hiero?view=week).
+---
 
-## Code of Conduct
+## Details
+
+**Organization:** Hiero TSC  
+**Date:** July 1, 2025  
+**Time:** 10:00am ET  
+**Recording:** [Zoom Recording](https://zoom.us/rec/share/WTclNCGE39ghAocfLX99EXAO0bix-dJQ4u3_xQ9smm10y6znAI0l6dnggLMNOGTK.V5Ea3cYrmms7vgv7)
+
+---
+
+## TSC Members
```

Comment:
Can I request that this title is kept as "TSC Atendees"?
I am working on automating attendance for the quorum and vote validation and my script will be simpler if I can make these files consistent. 

For reference: https://github.com/hiero-ledger/tsc/pull/166/files